.github/dependabot.yml

```diff
@@ -0,0 +1,11 @@
+---
+version: 2
+updates:
+  - package-ecosystem: github-actions
+    directory: "/"
+    schedule:
+      interval: "weekly"
```

Comment:
Similarly for GitHub Actions, our strategy is to run this monthly, and in our workflow files, we pin actions at `v1`, `v2`, etc rather than `vX.Y.Z` to reduce noise.